coral-service/src/main/java/com/linkedin/coral/coralservice/utils/VisualizationUtils.java

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2022-2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.coralservice.utils;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.UUID;
+
+import org.apache.calcite.rel.RelNode;
+import org.apache.calcite.sql.SqlNode;
+
+import com.linkedin.coral.hive.hive2rel.HiveToRelConverter;
+import com.linkedin.coral.incremental.RelNodeIncrementalTransformer;
+import com.linkedin.coral.transformers.CoralRelToSqlNodeConverter;
+import com.linkedin.coral.trino.trino2rel.TrinoToRelConverter;
+import com.linkedin.coral.vis.VisualizationUtil;
+
+import static com.linkedin.coral.coralservice.utils.CoralProvider.*;
+
+
+public class VisualizationUtils {
+
+  public static File getImageDir() {
+    return new File(System.getProperty("java.io.tmpdir") + "/images" + UUID.randomUUID());
+  }
+
+  public boolean isValidFromLanguage(String fromLanguage) {
+    return fromLanguage.equalsIgnoreCase("trino") || fromLanguage.equalsIgnoreCase("hive");
```

Comment:
Add Spark since it is similar to hive?

Reply:
Currently, the coral service translation doesn't support spark as a source language either. Also is there way to translate a spark query w/o having to explicitly pass in a db? https://github.com/linkedin/coral/blob/db5606964bd19cb6deacb97fafc58c3945a8dea3/coral-spark/src/test/java/com/linkedin/coral/spark/TestUtils.java#L252C28-L252C28